src/bundler.ts

```diff
@@ -300,37 +310,53 @@ export class Bundler {
       reachedImports.add(resolvedUrl);
       this.rewriteImportedUrls(importDoc, resolvedUrl, docUrl);
 
-      let importParent: ASTNode;
       // TODO(usergenic): remove the remove() call when PolymerLabs/dom5#35 is
```

Comment:
looks like there was an update to #35, still needed? https://github.com/Polymer/dom5/issues/35